### DISCUSSION TITLE
Admin rights to run in Silent mode

### DISCUSSION THREAD

**Discussion by ravja** [Micke-K/IntuneManagement#161] [Q&A]
Good morning,
I have a problem when I launch the export in Silent mode from a VM where I do not have admin rights it gives me an error when initializing IntuneTools as well as when using MSAL dll
If I run the same command from the same VM with Admin rights I don't get this error.
Is it necessary to have admin rights on the VM where the script is launched or is there a way to do it without admin rights?

And I have another question regarding App dependencies, do you plan to integrate it soon or not?

Thanks for your help
Best regards

Gabriel
![Command](https://github.com/Micke-K/IntuneManagement/assets/70326541/93c7c8c2-7bc7-440f-87bc-4c1c2a13e509)

![error](https://github.com/Micke-K/IntuneManagement/assets/70326541/a0c22651-2a30-4c34-9110-9e6310607840)

**Comment by Micke-K** [comment]
Hello!

Is PowerShell in Constrained Language mode or are there some other restrictions for the normal user? The user will need write permission on the cs folder. 

Cheers!

**Reply by ravja** [reply]
Hi Mikael,
Thanks for your answer I have check the LanguageMode and I'm on FullLanguage Mode.
What you mean with CS Folder?

**Reply by Micke-K** [reply]
Hello !

So sorry. I missed replying to this. 

The is a subfoler call CS where CS files are stored. The svript will compile them on the fly and this requires write permission on the folder where the CS files are located. 

Cheers!

**Comment by ravja** [comment]
Hello,

I have checked the rights and the user used to run it as full rights on all folders in IntuneManagement-master.
The error still .

Best regards
Gabriel
